pkg/udev/common.go

```diff
@@ -143,9 +146,22 @@ func (device *UdevDevice) GetDevLinks() map[string][]string {
 	byIdLink := make([]string, 0)
 	byPathLink := make([]string, 0)
 	for _, link := range strings.Split(device.GetPropertyValue(UDEV_DEVLINKS), " ") {
+		/*
+			devlink is like - /dev/disk/by-id/scsi-0Google_PersistentDisk_demo-disk
+			parts = ["", "dev", "disk", "by-id", "scsi-0Google_PersistentDisk_demo-disk"]
+			parts[4] containis link index like model or wwn or sysPath (wwn-0x5000c5009e3a8d2b) (ata-ST500LM021-1KJ152_W6HFGR)
+		*/
 		parts := strings.Split(link, "/")
 		if util.Contains(parts, BY_ID_LINK) {
-			byIdLink = append(byIdLink, link)
+			/*
+				By default one by-id link which contains serial number and bus is created so we put that link as
+				1st entry of by-id links it contains bus at prefix and - separated vendor, model, serial at suffix
+			*/
```

Comment:
Please modify the above text as follows:
A default by-id link is observed to be created for all types of disks (physical, virtual and cloud). This link has the format - bus, vendor, model, serial - all appended in the same order. Keeping this link as the first element of array for consistency purposes.

Reply:
done.